### ISSUE TITLE
`export` does not work

### ISSUE THREAD

**Issue by heitorPB** [hackmdio/codimd-cli#6]
The import functions works nicely! But the export doesn't. I tried all combinations.

```
$ env CODIMD_SERVER='https://demo.codimd.org' ./codimd import /tmp/out.md
Found. Redirecting to https://demo.codimd.org/BEy9CA7aTr2blBPAoAXDmg

$ env CODIMD_SERVER='https://demo.codimd.org' ./codimd export --slides BEy9CA7aTr2blBPAoAXDmg out.zip
Cannot specify both -k or --convert-file-only and -O if multiple URLs are given, or in combination
with -p or -r. See the manual for details.

$ env CODIMD_SERVER='https://demo.codimd.org' ./codimd export --html BEy9CA7aTr2blBPAoAXDmg out.html
--2018-09-26 17:01:58--  https://demo.codimd.org/Found.%20Redirecting%20to%20https://demo.codimd.org/s/Bkn2szKYX
Loaded CA certificate '/etc/ssl/certs/ca-certificates.crt'
Resolving demo.codimd.org (demo.codimd.org)... 2606:4700:30::681b:92b6, 2606:4700:30::681b:93b6, 104.27.147.182, ...
Connecting to demo.codimd.org (demo.codimd.org)|2606:4700:30::681b:92b6|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2018-09-26 17:01:58 ERROR 404: Not Found.

Converted links in 0 files in 0 seconds.
```

There's a typo in `codimd:38`, it should be `..."$CODIMD_SERVER/$publish_url"` right? With the slash? And I removed the `-O "$3"` option of the wget to make it "work".

**Comment by B1Sandmann** [comment]
You need to specify a output file to make it work with the `-O "$3"` then it should work. I proposed a way to fix this in #8  but there are probably nicer ways.

**Comment by pirate** [comment]
Merged #8 to fix this.
